webapp/tbapi/tbapi/blueprints/api.py

```diff
@@ -1,20 +1,137 @@
-from flask import Blueprint, render_template, make_response, jsonify, flash, redirect, url_for, request, current_app
+from flask import Flask, Blueprint, render_template, make_response, jsonify, flash, redirect, url_for, request, current_app
 from tbapi.models.mpower import Patient, User
+from flask_swagger import swagger
+from tbapi.models.tbapp import Note
 
 bp = Blueprint('api', __name__)
 
-@bp.route('/patients')
-def patient_index():
-    return jsonify(patients=[i.serialize for i in Patienst.query.all()])
+@bp.route('/notes/<int:id>', methods=['GET']) # note id
```

Comment:
I would suggest splitting this view function, so each route is served by a separate view function. It should make error handling much easier.

Reply:
done